src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/InstanceRepository.cs

```diff
@@ -62,234 +57,155 @@ public InstanceRepository(IOptions<AzureCosmosSettings> cosmosettings)
             _client.OpenAsync();
         }
 
-        /// <summary>
-        /// To insert new instance into instance collection
-        /// </summary>
-        /// <param name="item">the form data</param>
-        /// <returns>The deserialized formdata saved to file</returns>
-        public async Task<string> InsertInstanceIntoCollectionAsync(Instance item)
+        /// <inheritdoc/>
+        public async Task<Instance> Create(Instance instance)
```

Comment:
Is exception handling covered by another part of the code?

Reply:
Controller handles exceptions